visual-js/visual/src/cli/build.ts

```diff
@@ -56,9 +56,11 @@ const getBuildResults = async (
   api: VisualApi,
   { buildId, customId }: { buildId?: string; customId?: string },
 ): Promise<Record<string, number>> => {
-  const build = customId
-    ? await api.buildWithDiffsByCustomId(customId)
-    : await api.buildWithDiffs(buildId);
+  let build;
+
+  if (customId) build = await api.buildWithDiffsByCustomId(customId);
+  if (buildId) build = await api.buildWithDiffs(buildId);
```

Comment:
Not for type checking, no, since `buildId` _could_ be undefined just like the Cypress types which would be unacceptable for the API after updating / fixing the types for codegen.